yoshi/yoshi.py

```diff
@@ -0,0 +1,72 @@
+import numpy as np
+from astropy.table import Table
+from chandra_aca.transform import calc_aca_from_targ
+from chandra_aca.drift import get_aca_offsets
+import proseco
+
+
+def run_one_yoshi(**kw):
+    """
+    Run proseco and sparkles for an observation request in a roll/temperature/man_angle
+    scenario.
+
+    :param detector: detector (ACIS-I|ACIS-S|HRC-I|HRC-S)
+    :param chipx: chipx from zero-offset aimpoint table entry for obsid
+    :param chipy: chipy from zero-offset aimpoint table entry for obsid
+    :param chip_id: chip_id from zero-offset aimpoint table entry for obsid
+    :param ra_targ: target RA (degrees)
+    :param dec_targ: target Dec (degrees)
+    :param roll_targ: target Roll (degrees)
+    :param offset_y: target offset_y (arcmin)
+    :param offset_z: target offset_z (arcmin)
+    :param sim_offset: SIM Z offset (steps)
+    :param focus_offset: SIM focus offset (steps)
+    :param dither_y: Y amplitude dither (arcsec)
+    :param dither_z: Z amplitude dither (arcsec)
+    :param obs_date: observation date (for proper motion and ACA offset projection)
+    :param t_ccd: ACA CCD temperature (degrees C)
+    :param man_angle: maneuver angle (degrees)
+    :returns: dictionary of (ra_pnt, dec_pnt, roll_pnt,
+                             N_critical, N_warning, N_caution, N_info,
+                             P2, guide_count)
+
+    """
+
+    # Calculate dynamic offsets using the supplied temperature.
+    aca_offset_y, aca_offset_z = get_aca_offsets(kw['detector'],
+                                                 kw['chip_id'], kw['chipx'], kw['chipy'],
+                                                 kw['obs_date'], kw['t_ccd'])
+
+    # Get the ACA quaternion using target offsets and dynamic offsets.
+    # Note that calc_aca_from_targ expects target offsets in degrees and obs is now in arcmin
+    q_pnt = calc_aca_from_targ((kw['ra_targ'], kw['dec_targ'], kw['roll_targ']),
+                               (kw['offset_y'] / 60.) + (aca_offset_y / 3600.),
+                               (kw['offset_z'] / 60.) + (aca_offset_z / 3600.))
+
+    # Run proseco and sparkles
+    aca = proseco.get_aca_catalog(obsid=0,
```

Comment:
I just saw it as a no-op.  Would you like an option to return the catalogs so there'd be something coming back that would use the obsid?

Reply:
Doesn't really matter, leave it.

Reply:
Not sure which way would be "leave it" now? (had already set to take and use obsid even if it doesn't do anything).  But I suppose now that I have explicit required args in the function definition, taking and using obsid makes a lot of sense anyway, as it seems highly likely it will be in the source data row (like my example job) and it would be annoying to get 

```
run_one_yoshi() got an unexpected keyword argument 'obsid'
```